client/src/model/dto/BoardCreation.tsx

```diff
@@ -0,0 +1,6 @@
+interface BoardCreation {
```

Comment:
```suggestion
export interface BoardCreation {
```
and remove export {} 

same for others